postgresql/cluster/hot-standby/slave.sls

```diff
@@ -0,0 +1,74 @@
+{%- from "postgresql/map.jinja" import cluster with context %}
+
+{%- if cluster.enabled %}
+
+postgresql_slave_service_stop:
```

Comment:
I used the same unless condition that is used throughout the file, it will only stop the service during the first run